config/settings.yml

```diff
@@ -1444,6 +1444,14 @@ github_cvu:
   integration_id: ~
   private_pem: ~
 
+chip:
+  url: https://vpce-06399548ef94bdb41-lk4qp2nd.execute-api.us-gov-west-1.vpce.amazonaws.com
+  base_path: dev
+  tmp_api_user: TzY6DFrnjPE8dwxUMbFf9HjbFqRim2MgXpMpMciXJFVohyURUJAc7W99rpFzhfh2B3sVnn4
+  tmp_api_id: 2dcdrrn5zc
+  tmp_api_username: vetsapiTempUser
+  mock: true
```

Comment:
Just confirming none of these should be stored in Param store

Reply:
Yes, it is not required for test environment